src/components/ui/breadcrumb.tsx

```
@@ -0,0 +1,90 @@
+}
+
+function BreadcrumbPage({ className, ...props }: React.ComponentProps<'span'>) {
+    return (
+        <span
+            data-slot="breadcrumb-page"
+            role="link"
+            aria-disabled="true"
+            aria-current="page"
+            className={cn('text-foreground font-normal', className)}
+            {...props}
+        />
```

Biome (2.4.7)

[error] The HTML element with the interactive role "link" is not focusable.

(lint/a11y/useFocusableInteractive)